### ISSUE TITLE
Conformance view broken

### ISSUE THREAD

**Issue by mattgodbolt** [mattgodbolt/compiler-explorer#535]
The following link: https://godbolt.org/g/ehKEau shows all compilers build OK. But if you look at individual output, some compilers fail.

I just updated and then suddenly the compilers failed... hmm

Repro steps:
* New CE window
* Reset UI
* Type in the code from below
* close compiler window
* create conformance view
* click to add three compilers - leave them blank
* paste `-Wuseless-cast -std=c++1z` options into each options slow
* pick x86_64 gcc 6.3 as first compiler - builds OK
* pick x86 gcc 7.2 as second compiler - builds OK -- BUT should fail (options make it fail)
* pick x86 gcc trunk as third compiler - builds OK - BUT should fail as above
* Now create a compiler window and notice suddenly the 7.2 and trunk compilers start working

(code: 
```
enum class Enum : char { A = 0, B = 1 };
struct S {
  Enum e { Enum::A };
};
```
)

**Comment by mattgodbolt** [comment]
May require a drag of the conformance view, not a click (couldn't repro when I clicked it)

**Comment by RabsRincon** [comment]
This has uncovered a nasty problem that might be present elsewhere. the configs passed to GoldenLayout have values for when they were declared, not called, which makes a lot of sense when you think about it but I somehow didn't realize until now.
Hopefuly I can fix it today, tops

**Comment by RabsRincon** [comment]
And as always, I forgot to close by commit. Closed by c4d6f1a
